src/controller.ts

```diff
@@ -1,9 +1,10 @@
 import { Application } from "pixi.js";
 import { Minigame, MinigameDelegate } from "./minigames/minigame";
 import { KeyboardMinigame } from "./minigames/keyboardMinigame";
+import { RhythmMinigame } from "./minigames/rhythmMinigame";
 import { Interlude, InterludeDelegate } from "./interlude/interlude";
 
-const ALL_MINIGAMES = [Minigame, KeyboardMinigame];
+const ALL_MINIGAMES = [RhythmMinigame];
```

Comment:
Just do the thing i suggested of actually ending the minigame and then add rhythmMinigame at the end of the list.